ecommerce/views_test.py

```diff
@@ -1227,6 +1229,42 @@ def test_products_viewset_list(user_drf_client, coupon_product_ids):
         )
 
 
+def test_products_viewset_list_ordering(user_drf_client):
+    """
+    Test that the ProductViewSet returns all products ordered alphabetically
+    by course/program title
+    """
+    programs = ProgramFactory.create_batch(
+        2, title=factory.Iterator(["Z Program", "A Program"])
+    )
+    courses = CourseFactory.create_batch(
+        2, title=factory.Iterator(["Z Course", "A Course"])
+    )
+    runs = CourseRunFactory.create_batch(2, course=factory.Iterator(courses))
```

Comment:
Looks great. Just one bit of cleanup:

```suggestion
    runs = CourseRunFactory.create_batch(2, course__title=factory.Iterator(["Z Course", "A Course"]))
```

Reply:
Incorporated & ready for review.